Adeunis/decoder.js

```diff
@@ -0,0 +1,102 @@
+// TTN decoder function for the Adeunis Field Test Device LoRaWAN
```

Comment:
I think we should indicate in the filename that this is only for the 868 to avoid any confusion.

Reply:
Agreed, Filename changed 868mhz decoder.js